pinot-tools/pom.xml

```diff
@@ -34,6 +34,8 @@
   <properties>
     <pinot.root>${basedir}/..</pinot.root>
     <aws.version>2.14.28</aws.version>
+    <scala.version>2.12</scala.version>
```

Comment:
Once you start pulling in Scala code, you need to ensure that every dependency that's also using Scala is using the same version. So I think this should go in the top-level `pom.xml` file. Also note that pinot-kafka and pinot-spark have dependencies on Scala 2.11, which I believe will cause runtime problems if they are on the classpath when the tool is being run (and it's using 2.12).

Reply:
I have actually excluded the scala code from plugin. It is just that the plugin requires it in name.